package.json

```diff
@@ -19,28 +19,29 @@
   "scripts": {
     "prepush": "npm test",
     "prebuild": "del-cli ./dist",
-    "build": "babel src -d dist",
+    "build": "tsc",
     "lint": "eslint src/**/*.js",
     "test": "npm run lint --silent && npm run test:unit --silent",
     "test:unit": "cross-env NODE_ENV=test LOG4JS_LEVEL='OFF' jest --coverage",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "dependencies": {
     "chalk": "^2.1.0",
+    "express": "^4.16.2",
```

Comment:
Does `express` need to be a dependency? I believe it should only be a devDependency.